sources/packages/backend/workflow/test/2024-2025/parttime-assessment/awards-amounts/parttime-assessment-awards-amount-CSGD.e2e-spec.ts

```diff
@@ -50,7 +50,7 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CS
       calculatedAssessment.variables.dmnPartTimeAwardFamilySizeVariables
         .limitAwardCSGDIncomeCap,
     );
-    expect(calculatedAssessment.variables.federalAwardCSGDAmount).toBe(1260);
+    expect(calculatedAssessment.variables.federalAwardNetCSGDAmount).toBe(1260);
```

Comment:
Minor. `federalAwardCSGDAmount` is renamed to `federalAwardMaxCSGDAmount`. Hence an an assert to check

```ts
expect(calculatedAssessment.variables.federalAwardMaxCSGDAmount).toBe(1260);
```

Reply:
Please ignore this for hotfix PR. It can be accommodated in other PR to main.